packages/blade/src/components/Button/BaseButton/getBaseButtonStyles.ts

```diff
@@ -0,0 +1,31 @@
+import type { StyledBaseButtonProps } from './StyledBaseButton';
+
+const getBaseButtonStyles = ({
+  color,
+  hoverColor,
+  activeColor,
+}: Omit<StyledBaseButtonProps, 'children' | 'onClick'>): string => `
+  display:flex;
+  flex-direction: row;
```

Comment:
do we really need this? 🤔

Reply:
Ignore all the styles, they'll all change 😅